content/docs/languages/go/quickstart.md

```diff
@@ -216,7 +204,7 @@ from the `examples/helloworld` directory:
 - Explore the [API reference](../api).
 
 [Basics tutorial]: ../basics/
-[download]: https://github.com/grpc/grpc-go/archive/{{< param grpc_go_release_tag >}}.zip
+[download]: https://github.com/grpc/grpc-go/archive/master.zip
```

Comment:
The comment made for the change to the `git clone` command, applies here too -- https://github.com/grpc/grpc.io/pull/436/files#r499944443. I.e., this will need to be reverted eventually. See #437.